keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/docs/Description.kt

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spinnaker.keel.api.docs
+
+import kotlin.annotation.AnnotationTarget.CLASS
+import kotlin.annotation.AnnotationTarget.PROPERTY
+
+@Target(CLASS, PROPERTY)
+annotation class Description(val value: String)
```

Comment:
Our own annotation we can use to add a description to models in the API without relying on Swagger dependency